shared/versions/stack/master.asciidoc

```diff
@@ -11,7 +11,7 @@ bare_version never includes -alpha or -beta
 :minor-version:          9.0
 :major-version:          9.x
 :prev-major-version:     8.x
-:prev-major-last:        8.17
+:prev-major-last:        8.18
```

Comment:
@lcawl : this is the only change planned for `master.asciidoc`. If I have to leave it untouched and keep `prev-major-last` on 8.17 let me know and I'll do it. In such case... when should we update that line? because the previous major last will be 8.18